<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialConfigDialog</class>
 <widget class="QDialog" name="SerialConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>629</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Serial Configuration</string>
  </property>
  <widget class="QListWidget" name="listWidgetSerialDevices">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>641</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonRefreshDevices">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelConnectedDevices">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>20</y>
     <width>261</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Connected devices</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>447</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBoxBaudRate">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <item>
       <property name="text">
        <string>1200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4800</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>19200</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>38400</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>57600</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>115200</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="QLabel" name="labelReadBufferSize">
      <property name="text">
       <string>Read Buffer Size</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="labelDataBits">
      <property name="text">
       <string>Data Bits</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="labelFlowControl">
      <property name="text">
       <string>Flow Control</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QComboBox" name="comboBoxFlowControl">
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hardware</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Software</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="QSpinBox" name="spinBoxBufferSize"/>
    </item>
    <item row="1" column="2">
     <widget class="QComboBox" name="comboBoxDataBits">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>7</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Parity</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="labelBaudRate">
      <property name="text">
       <string>Baudrate</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QComboBox" name="comboBoxParity">
      <item>
       <property name="text">
        <string>None</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Even</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Odd</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Space</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Mark</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLabel" name="labelStopBits">
      <property name="text">
       <string>Stop Bits</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignBottom|Qt::AlignHCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="6">
     <widget class="QComboBox" name="comboBoxStopBits">
      <item>
       <property name="text">
        <string>One</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>One And Half</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Two</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>390</y>
     <width>168</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButtonExitOk">
      <property name="text">
       <string>Ok</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonExitCancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
